src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/memory.cpp

```diff
@@ -65,7 +65,6 @@ TEST_P(MemoryTestGna, CompareWithRefs) {
 std::vector<ngraph::helpers::MemoryTransformation> transformation{
     ngraph::helpers::MemoryTransformation::NONE,
     ngraph::helpers::MemoryTransformation::LOW_LATENCY_V2,
-    ngraph::helpers::MemoryTransformation::LOW_LATENCY_V2_REGULAR_API,
```

Comment:
Add the TODO:CVS-125596 ?

Reply:
no, it is not the same issue. LOW_LATENCY_V2_REGULAR_API is removed from all plugin tests.